jedi_language_server/initialization_options.py

```diff
@@ -38,6 +38,7 @@ class Completion(Model):
 
 class Diagnostics(Model):
     enable: bool = True
+    did_close: bool = True
```

Comment:
Per your recommendation [here](https://github.com/pappasam/jedi-language-server/issues/153#issuecomment-875844190), we will want to remove the `did_close` option from `initialization_options.py` and instead rely on the already-existing `enable` option. Eg, we'll always clear diagnostics as long as diagnostics are enabled by the user. In short, I recommend that we remove this option.

Reply:
thanks, I removed the option.